data/binding/mapbinding.go

```diff
@@ -296,23 +297,23 @@ func (b *boundStruct) Reload() (retErr error) {
 			continue
 		}
 
-		var err error
+		var r error
```

Comment:
I don't think `r` is an obvious short name for an error

Reply:
I'm pretty sure it is used all over the place if `err` is already taken and the scope does not call for a more specific name.

But...  what would you use?

Reply:
I don't think we have a standard as it does not happen very often, but as most single rune variables use the first letter I would have thought `e` better suits? Or `newErr` if you want to follow the idea the original code had with `retErr`.